File: bench/fib.rb
def fib(n)
  return n if n <= 1
  fib(n - 1) + fib(n - 2)
end

start = Time.now
result = fib(35)
elapsed = Time.now - start
puts "fib(35) = #{result}"
puts "Time: #{elapsed.round(4)}s"

File: bench/fib.py
import time

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

start = time.time()
result = fib(35)
elapsed = time.time() - start
print(f"fib(35) = {result}")
print(f"Time: {elapsed:.4f}s")
